storagenode/storagenodedb/consoledb.go

```diff
@@ -62,8 +66,8 @@ func (db *consoledb) GetSatelliteIDs(ctx context.Context, from, to time.Time) (_
 func (db *consoledb) GetDailyTotalBandwidthUsed(ctx context.Context, from, to time.Time) (_ []console.BandwidthUsed, err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	since, _ := getDateEdges(from.UTC())
-	_, before := getDateEdges(to.UTC())
+	since, _ := db.dateUtils.DayBoundary(from.UTC())
```

Comment:
`since, _ := date.DayBoundary(from.UTC())`

if the package name date is conflicting name it in the import to something like this:
`dateutils storj.io/storj/internal/date`
and then call it like this
`since, _ := dateutils.DayBoundary(from.UTC())`